.github/drafts/16506/16506_category_slider_outline.md

```diff
@@ -0,0 +1,62 @@
+*** Category Slider ***
+
+The Category Slider widget allows users to choose from a fixed set of incremental values.
+
+< TODO: NEED VIDEO DEMO >
+
+---
+# Usage
+- Example use case: Surveys, i.e. Strongly agree/ agree/ neutral/ disagree/ strongly disagree
+
+---
+
+# Properties
+
+Properties allow you to edit the widget, connect it with other widgets and customize the user actions.
+
+## Widget Properties
+
+These properties allow you to edit the Category Slider widget. The following table lists all the widget's unique properties; if you do not see a certain widget property here, it may be listed under [Common Widget Properties]().
+
+| **Property** | **Description** |
+|-----------|-----------------|
+| **Options** | An _array of objects_ with keys `value` and `label` which define the possible selections that the user can make with the slider. |
+| **Default Value** | This sets the slider's initial value, before the user makes any adjustments. This value should be between the slider's **Min Value** and **Max Value**. _(number)_ |
+| **Marks** | An _array of objects_ with keys `value` and `label` which define where reference labels should appear below the widget's slider. These labels can be shown or hidden with the **Show Marks** property toggle. |
+| **Tooltip Always On** | Hovering over the slider with the mouse cursor shows the slider's selected value in a tooltip; enabling this setting will force the tooltip to always be visible, regardless of the cursor's location. _(bool)_ |
```

Comment:
For the category `slider` we have removed the **Tooltip Always on** option. I would request you to please remove it from here.